models/TaskAssignee.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const TaskAssigneeSchema = new mongoose.Schema(
+	{
+		assigneeId: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+		assignedBy: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```

Comment:
IMO I don't see the need for `assignedBy`. We have that ref in the task schema. A task is created by the group head and this field will not change when the same task is assigned to several people.

Reply:
When fetching tasks for a user, assignedBy is used to fetch tasks created by a user whereas assigneeId is used to fetch tasks assigned to the user

Reply:
okay cool!